config/servicemappings/accesscontextmanager.yaml

```diff
@@ -356,3 +356,30 @@ spec:
                 group: resourcemanager.cnrm.cloud.google.com
               targetField: "number"
               valueTemplate: "projects/{{value}}"
+    - name: google_access_context_manager_service_perimeter_resource
+      kind: AccessContextManagerServicePerimeterResource
+      idTemplate: "{{perimeter_name}}/{{resource}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      v1alpha1ToV1beta1: true
+      storageVersion: v1alpha1
+      autoGenerated: false
+      resourceReferences:
+        - key: projectRef
```

Comment:
We probably want to call the field `resourceRef`. See bullet point 3 in this doc: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/certificatemanager.yaml#L61